lib/muchos/config/base.py

```diff
@@ -182,6 +183,10 @@ def _verify_config(self, action):
             if self.java_product_version() >= 11 and StrictVersion(self.version('accumulo').replace('-SNAPSHOT','')) <= StrictVersion("1.9.3"):
                 exit("ERROR - Java 11 is not supported with Accumulo version '{0}'".format(self.version('accumulo')))
 
+            # validate and fail if we are using ZooKeeper 3.5.5 or greater and Accumulo 1.9.x or less
+            if StrictVersion(self.version('zookeeper')) >= StrictVersion("3.5.5") and StrictVersion(self.version('accumulo').replace('-SNAPSHOT','')) < StrictVersion("1.10.0"):
```

Comment:
If it doesn't work with 1.9, I wouldn't expect it to work with 1.10 either... but perhaps that will be fixed before released.

Reply:
The issues are minor, possibly a good first issue for someone, so I am thinking they might get fixed. The zookeeper jar is now in a lib directory, so start-all.sh and general.classpaths have to be updated. (I am not sure whether general.classpaths should be updated in Accumulo itself or in Fluo-Muchos, but at least the ZK jar check in start-all.sh would need to be fixed.)

Reply:
@ctubbsii one of us will submit a PR for the start-all.sh script early next week.